pyproject.toml

```diff
@@ -32,6 +32,16 @@ boto3 = "*"
 minio = "^7.2.7"
 gevent = "^24.2.1"
 pyarrow = "16.1.0" # transitive dependency, pinned here to avoid conflicts
+openai = "*"
```

Comment:
Can you move these to the test dependencies? Since we will be using these inside the docker sandbox AND when running unit tests.

Also, can you add pip install of these libraries to the Dockerfile of our sandbox so that it is by default installed?